app/routes/goal.py

```diff
@@ -0,0 +1,125 @@
+from flask import abort, Blueprint, jsonify, make_response, request
+from app import db
+from app.models.goal import Goal
+from datetime import datetime
+from app.models.task import Task
+from .task import get_one_task_or_abort
+import requests
+import os
+
+goal_bp = Blueprint("goal", __name__, url_prefix="/goals")
+
+
+@goal_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+    try:
+        new_goal = Goal(title=request_body["title"])
+    except:
+        return abort(make_response({"details": "Invalid data"}, 400))
```

Comment:
Great use of a try-except for input validation.  You should probably also indicate what data is invalid.